app/models/user.rb

```diff
@@ -16,4 +16,9 @@ def member_of?(team)
   def full_name
     "#{first_name} #{last_name}"
   end
+
+  def confirm_email
+    self.email_confirmed_at = Time.current
+    save
```

Comment:
Why not simply
```suggestion
    update(email_confirmed_at: Time.current)
```
?

Reply:
Yap, could be. Replaced to use update and `DateTime.now`.